packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+const horizontalPadding = '30px';
+
+export const Root = styled.div`
+	width: 272px;
+	padding: 35px 0px;
+	> *:not( .components-navigation__menu ) {
+		margin-left: ${ horizontalPadding };
+		margin-right: ${ horizontalPadding };
```

Comment:
We can drop the `margin-right`.

Reply:
I think we'll need the margin-right to wider items from touching the side, but based on the other items discussed in this PR, I'm going to entirely remove this margin as I think it's too opinionated for the component.